lib/Doctrine/Common/Annotations/AnnotationReader.php

```diff
@@ -292,6 +292,38 @@ public function getMethodAnnotation(ReflectionMethod $method, $annotationName)
         return null;
     }
 
+    /**
+     * {@inheritDoc}
+     */
```

Comment:
I don't think any doc is being inherited from anywhere

Reply:
The methods were missing in the interface. Fixed.